manifests/p/ProgrammingUnlimited/FileVerifier++/0.6.6.6050/ProgrammingUnlimited.FileVerifier++.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: ProgrammingUnlimited.FileVerifier++
+PackageVersion: 0.6.6.6050
+InstallerLocale: en-US
+InstallerType: wix
+ProductCode: '{9DEB845B-3F05-4EFF-A161-1047BD2749B6}'
+Installers:
+- InstallerUrl: https://yer.dl.sourceforge.net/project/fileverifier/fileverifier/0.6.6.6050/fv-0.6.6.6050W-x86_64.msi

```

```suggestion
- InstallerUrl: https://dl.sourceforge.net/project/fileverifier/fileverifier/0.6.6.6050/fv-0.6.6.6050W-x86_64.msi
```
I believe that omitting the first subdomain allows their CDN to redirect to their nearest endpoint.